package.json

```diff
@@ -309,7 +309,7 @@
     "browser-stdout": "1.3.0",
     "commander": "2.11.0",
     "debug": "3.1.0",
-    "diff": "3.3.1",
+    "diff": "3.4.0",
```

Comment:
why?

Reply:
Just to keep dependencies actual. The [changes] made between versions do not influence the functions used by us, but the fresh version of library will simplify the analysis of changes if such appear and they will influence our code

[changes]: https://github.com/kpdecker/jsdiff/compare/v3.3.1...v3.4.0